internal/buildscripts/packaging/fpm/etc/otel/collector/splunk-support-bundle.sh

```diff
@@ -125,8 +125,8 @@ getConfig() {
     # Also need to get config in memory as dynamic config may modify stored config
     # It's possible user has disabled collecting in memory config
     if timeout 1 bash -c 'cat < /dev/null > /dev/tcp/localhost/55554'; then
-        curl -s http://localhost:55554/debug/configz/initial >"$TMPDIR"/config/initial.yaml 2>&1
-        curl -s http://localhost:55554/debug/configz/effective >"$TMPDIR"/config/effective.yaml 2>&1
+        wget -q http://localhost:55554/debug/configz/initial -O "$TMPDIR"/config/initial.yaml 2>&1
```

Comment:
I think any support changes like this are potentially too impacting for customer envs. imo the client executable should be invoked w/ a variable based on what's available. if we are going to require one, we should include it as a dep in our packages.